specs/test/testplan.md

```diff
@@ -109,6 +102,26 @@ It should cover the basic function to pass, to fail, and error conditions.
 * note
 * automated?
 ---
+
+### smartcontract
+
+* test case # : SC1
+* description : deploy ERC20 smart contract in 1 shard
+* test procedure : smart contract is deployed in a automated fashion e.g. faucet contract
```

Comment:
faucet contract is not ERC20, right?
we need to create an ERC20 contract

Reply:
Fixed.